spec/integration/_dev/benchmark/system.scenario.spec.yml

```diff
@@ -0,0 +1,65 @@
+spec:
+  type: object
+  additionalProperties: false
+  required:
+    - data_stream
+    - input
+  properties:
+    package:
+      description: The package to benchmark.
+      type: string
+    description:
+      description: A description of the system benchmark scenario.
+      type: string
+    version:
+      $ref: "../../manifest.spec.yml#/definitions/version"
+    input:
+      description: The input of the package to benchmark.
+      type: string
+    vars:
+      description: The package level variables.
+      type:
+        - object
+        - "null"
+      additionalProperties: true
+      default: ~
+    data_stream:
+      description: Data-stream parameters.
+      type: object
+      additionalProperties: false
+      required:
+        - name
+      properties:
+        name:
+          description: The name of the data_stream to benchmark.
+          type: string
+        vars:
+          description: The data_stream specific variables.
+          type:
+            - object
+            - "null"
+          additionalProperties: true
+          default: ~
+    warmup_time_period:
+      description: |
+        A time period as a duration string for warmup of the benchmark. 
+        All response data captured during warmup will not show up in the measurement results.
+        By default the value is 0 seconds, which means that all data captured will be taken into account for the benchmark results.
+      type: string
+      default: ""
+      example: 10s
+    benchmark_time_period:
+      description: |
+        A time period as a duration string for the duration of the benchmark.
+        By default the value is 0 seconds, which means that the benchmark will run until no more data is ingested.
+      type: string
+      default: ""
+      example: 10m
+    wait_for_data_timeout:
+      description: Timeout for waiting for data during a system benchmark.
+      type: string
+      default: 10m
+      example: 10m
+    corpora:
+      type: object
+      $ref: "../corpora.spec.yml#/properties/corpora"
```

Comment:
> To keep this, I would move this `corpora` object definition here in this file under the `definitions` key. And later, it can be referenced here and it can be referenced in the `rally.benchmark.spec.yml` file too

👍 on this

I knew the reference to `properties` was wrong and it should rather be to `definitions`, but looking quickly through the json schema docs I could not find the exact way for it